src/hub/related_models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .hub_metadata import HubMetadata
+from .hub_provider import HubProvider
+from .hub_v2 import HubV2
```

Comment:
These imports need to be in `models.py` as well.

I know this is a bit out of scope for the project, but if you could separate out the current models in `models.py` into their respective files, that would be great too.